airflow/jobs/backfill_job.py

```diff
@@ -689,6 +688,9 @@ def tabulate_tis_set(set_tis: Set[TaskInstance]) -> str:
 
         return err
 
+    def _get_dag_with_subdags(self):
+        return [self.dag] + self.dag.subdags
+
```

Comment:
```suggestion
    def _get_dag_with_subdags(self):
        return [self.dag] + self.dag.subdags
```

Keep it the way it was -- adding a one line function that is only called in one place doesn't make readability any better.

Reply:
It may help in future refactorings but sure, reverted :)

Reply:
YAGNI is my general rule -- keep the change as small as sensible for now, if we need it in the future then we do it then.